Formula/poac.rb

```diff
@@ -0,0 +1,34 @@
+class Poac < Formula
+  desc "Package manager for C++"
+  homepage "https://github.com/poacpm/poac"
+  url "https://github.com/poacpm/poac/archive/refs/tags/0.3.4.tar.gz"
+  sha256 "972727bd1f45f9cb1a0aae1994533fae75234614ffe5df2a316690698db9dd74"
+  license "Apache-2.0"
+  head "https://github.com/poacpm/poac.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on macos: :big_sur # C++20
+  depends_on "openssl@3"
```

Comment:
We should also use Homebrew versions of these packages (where available): https://github.com/poacpm/poac#build-from-source

See: https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae

Reply:
Sure :)